src/routes/associations.js

```diff
@@ -20,13 +20,19 @@ module.exports = function (app, model, Implementation, integrator, opts) {
   }
 
   function getAssociation(request) {
+    const pathSplit = request.route.path.split('/');
+    let associationName = pathSplit[pathSplit.length - 1];
+
+    if (['.csv', 'count'].includes(associationName)) {
+      associationName = pathSplit[pathSplit.length - 2];
```

Comment:
Are you sure you didn't break the CSV export here?

Reply:
Thanks, fixed ! 🙈